packages/components/src/sign-up/login-widget/LoginInfo.tsx

```diff
@@ -0,0 +1,78 @@
+"use client";
+
+import type { zSignInValues } from "@good-dog/trpc/schema";
+import { useFormContext } from "react-hook-form";
+import type { z } from "zod";
+import RHFTextInput from "../../rhf-base/RHFTextInput";
+import { Checkbox } from "@good-dog/ui/checkbox";
+import Link from "next/link";
+import { Button } from "@good-dog/ui/button";
+
+interface LoginInfoProps {
+  onVerifyCredentials: () => void;
+}
+
+type LoginFormFields = z.input<typeof zSignInValues>;
+
+export default function LoginInfo({ onVerifyCredentials }: LoginInfoProps) {
+  const {
+    formState: { errors },
+  } = useFormContext<LoginFormFields>();
+
+  return (
+    <form
+      className="pt-[70px] pr-[40px]"
+      onSubmit={(e) => {
+        e.preventDefault();
+        onVerifyCredentials();
+      }}
+    >
+      <h1 className="text-sign-up-header font-medium">Welcome back!</h1>
+      <h3 className="text-base-label">All fields below are required</h3>
+      <div className="pt-[32px]">
+        <RHFTextInput<LoginFormFields>
+          rhfName={"email"}
+          label={"Email"}
+          placeholder={"example@gmail.com"}
+          id="email"
+          errorText={errors.email ? errors.email.message : undefined}
+          required={true}
+        ></RHFTextInput>
```

Comment:
```suggestion
        />
```
nitpick. Same thing for the field below.